ui/core/sim_ui.ts

```diff
@@ -224,7 +224,11 @@ export abstract class SimUI<SpecType extends Spec> {
 				
         const linkUrl = new URL(window.location.href);
         linkUrl.hash = encoded;
-				navigator.clipboard.writeText(linkUrl.toString());
+        if (navigator.clipboard == undefined) {
+          console.log("LINK:", linkUrl.toString());
```

Comment:
I think an alert would be better than logging to console in this case.

Reply:
:+1:  This was a weird case where using an http:// address that wasn't localhost couldn't get access to clipboard?